app/assets/javascripts/group-assignment-invitations.coffee

```diff
@@ -10,6 +10,11 @@ $('.group_assignment_invitations.show').ready ->
       $submitButton.prop('disabled', true)
   )
 
+  $('form').submit( ->
+    hiddenField = $(this).find('#student_identifier_hidden')
+    hiddenField.val($('#student_identifier').val())
+  )
```

Comment:
Explanation on this:

In the group assignment invitation page, we have multiple forms, and students will choose either one of them. See the image below:

![screen shot 2016-06-06 at 9 24 09 am](https://cloud.githubusercontent.com/assets/1311594/15823539/131eab8c-2bc9-11e6-97d2-c6c7935a7fc3.png)

We will need to ask students to fill in their identifier when accepting the assignment. Since we have multiple forms on the same page, they have to share the same input field.

So I created a hidden input field for each form, and one stand-alone visible input field to let the student input the identifier. When the form is submitted, copy the value from the visible identifier input field to the hidden identifier input field, then the identifier value will be included in the form data.

I believe we can come up with a better solution for this. It would be great if you can give me some feedback on this. Thanks! 😸 

/cc @tarebyte @jasonlong